spec/query/request/boosting/numeric_field_value_match_boost_function_spec.rb

```diff
@@ -1,7 +1,20 @@
 RSpec.describe Solr::Query::Request::Boosting::NumericFieldValueMatchBoostFunction do
   describe '.to_solr_s' do
+    before do
+      Solr.configure do |config|
+        config.define_core(name: :'test-core') do |f|
+          f.field :machine_type
+        end
+      end
+    end
+
+    after(:each) do
```

Comment:
Isn't `:each` the default scope? I think it would be better to remove `(:each)` from here or add `(:each)` on `before` just to keep it consistent.